src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -10,15 +10,16 @@
                            style="vertical-align: text-bottom;"></info-icon>
             </span>
         </div>
-        <button v-if="outOfDateMessage" class="btn btn-white ml-2" @click="refresh">Refresh</button>
+        <button v-if="outOfDateMessage" class="btn btn-white ml-2" @click="refresh" v-translate="'refresh'"></button>
         <button class="btn btn-red" :class="selectedDataset && 'ml-2'" @click="toggleModal">{{ selectText }}</button>
         <modal id="dataset" :open="open">
-            <h4 v-if="!loading">Browse ADR</h4>
-            <p v-if="loading">Importing files - this may take several minutes. Please do not close your browser.</p>
+            <h4 v-if="!loading" v-translate="'browseADR'"></h4>
+            <p v-if="loading" v-translate="'importingFiles'"></p>
             <div v-if="!loading">
                 <tree-select :multiple="false"
                              :searchable="true"
                              :options="datasetOptions"
+                             :placeholder="select"
```

Comment:
I tried using `v-translate:placeholder="'select'"` here but it didn't show up in the browser for some reason.

Reply:
You might have to include it as a computed property. See how the ModelCalibrate component provides translations to DynamicForm e,g, https://github.com/mrc-ide/hint/blob/master/src/app/static/src/app/components/modelCalibrate/ModelCalibrate.vue#L12

Reply:
That's what I ended up doing with this. See the 'select' computed property,

Reply:
>I tried using v-translate:placeholder="'select'" here but it didn't show up in the browser for some reason.

For reference, that's because `placeholder` here is a custom prop being passed to the tree-select component, rather than the native HTML attribute of the same name.